packages/cli/src/commands/reserve/transfergold.ts

```diff
@@ -0,0 +1,55 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class TransferGold extends BaseCommand {
+  static description = 'Transfers reserve gold to other reserve address'
+
+  static flags = {
+    ...BaseCommand.flags,
+    value: flags.string({ required: true, description: 'The unit amount of Celo Gold (cGLD)' }),
+    to: Flags.address({ required: true, description: 'Receiving address' }),
+    from: Flags.address({ required: true, description: "Sender's address" }),
```

Comment:
This is the "spender's" address, yeah?

Reply:
ya...well technically they are also "sending" the transaction, which is why I keep messing this up.  I'll update